R/mainTests.r

```diff
@@ -1498,6 +1953,39 @@ filter_group_duplicate <- function(meta) {
   return(output)
 }
 
+# whitespace_filters -------------------------------------
+# check no filter labels have leading or trailing whitespace.
+
+whitespace_filters <- function(data, meta) {
+  filters <- meta %>%
+    filter(col_type == "Filter") %>%
+    pull(col_name)
+
+  test <- data %>%
+    select(all_of(filters), any_of(as.character(geography_matrix[, 2]))) %>%
+    gather(., "filter", "filter_label") %>%
```

Comment:
I think `gather()` is out of date and we should use `pivot_longer()` now? 

Either way, it's simple but I like how nifty this bit of code is, sets it up really nicely to pull out the detail for the error message 😀

Reply:
Moved over to pivot_now - much neater as it removes the NAs from the values! Hadn't seen this one before - thank you!